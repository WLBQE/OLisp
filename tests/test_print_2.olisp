(print 1)
(print 1.5)
(print true)
(print "Hello world!\nHello OLisp!")
(print "Test passed!")
