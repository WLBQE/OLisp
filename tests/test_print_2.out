1
1.5
true
Hello world!
Hello OLisp!
